src/python/turicreate/test/test_text_classifier.py

```diff
@@ -160,6 +160,20 @@ def test_sentiment_create_string_target(self):
         model = tc.text_classifier.create(data_str, target='rating')
         self.assertTrue(isinstance(model, tc.text_classifier.TextClassifier))
 
+    def test_invalid_data_set(self):
+        # infer dtype str
+        a = tc.SArray(['str', None])
+        b = tc.SArray(['str', 'str'])
+        # target contains none
+        sf = tc.SFrame({'a': a, 'b': b})
+        with self.assertRaises(ToolkitError):
+            tc.text_classifier.create(sf, target='a', features=['b'], word_count_threshold=1)
+        # feature contains none, #2402
```

Comment:
It might not be clear that `#2402` is referring to a GitHub issue. Please clarify.